### PR TITLE
repquota: add page

### PR DESCRIPTION
**Pull request by pearcem0**

### CODE REVIEW

**Comment by pearcem0** [comment]
Actually, I think this should go in pages/common rather than pages/linux.

**Comment by waldyrious** [comment]
Not sure. On OS X I get:

```
$ repquota -all
repquota: illegal option -- l
Usage:
	repquota [-v] [-g] [-u] -a
	repquota [-v] [-g] [-u] filesys ...
```

There are likely other inconsistencies as well. The version I have is:
`4.2 Berkeley Distribution       March 28, 2002       4.2 Berkeley Distribution`

Probably checking the BSD and GNU manpages for this and the other quota-related commands will reveal which can go on `common` and which need to be split into different platform versions.

**Comment by pearcem0** [comment]
You're right, this will need to be classed as Linux after all. Thanks

**Comment by pearcem0** [comment]
Hi, 
Is there anything else you need me to do for this pr?
Thanks,

File: pages/linux/repquota.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# repquota
+
+> Display a summary of existing file quotas for a filesystem.
+
+- Report on all existing quotas:
+
+`repquota -all`
+
+- Report  all quotas, even if there is no usage:
```
Comment (ostera) [inline pages/linux/repquota.md]: Double space between `Report` and `all`

File: pages/linux/repquota.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Report on quotas for users (default), only super-user may view quotas which are not their own:
+
+`repquota --user {{user}}`
+
+- Report on quotas for user groups:
+
+`repquota --group {{group}}`
+
+- Report on used space and limits in a human readable format:
```
Comment (ostera) [inline pages/linux/repquota.md]: I think elsewhere we've used `in human-readable format`. Saves a few bytes and maintains consistency. I'll search the repo and get back to you on this.

Comment (ostera) [inline pages/linux/repquota.md]: Yup: https://github.com/tldr-pages/tldr/search?utf8=%E2%9C%93&q=human-readable

Dashed.

**Comment by ostera** [comment]
Hey, just left a few comments :) other than that it looks good. Fix them and it has my :+1:

**Comment by pearcem0** [comment]
Thanks a lot. Made the changes.

File: pages/linux/repquota.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- Report on all existing quotas:
+
+`repquota -all`
+
+- Report all quotas, even if there is no usage:
+
+`repquota -v`
+
+- Report on quotas for users (default), only super-user may view quotas which are not their own:
```
Comment (waldyrious) [inline pages/linux/repquota.md]: This should be a semicolon, not a comma (see https://en.wikipedia.org/wiki/Run-on_sentence). Also, it's unclear what "for users (default)" means... care to clarify?

File: pages/linux/repquota.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# repquota
+
+> Display a summary of existing file quotas for a filesystem.
+
+- Report on all existing quotas:
+
+`repquota -all`
+
+- Report all quotas, even if there is no usage:
```
Comment (waldyrious) [inline pages/linux/repquota.md]: Do you mean "even if there are no quotas defined", or "even if quotas are defined, but not used" (not sure the latter actually makes sense)?

File: pages/linux/repquota.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+- Report on quotas for users (default), only super-user may view quotas which are not their own:
+
+`repquota --user {{user}}`
+
+- Report on quotas for user groups:
+
+`repquota --group {{group}}`
+
+- Report on used space and limits in a human-readable format:
```
Comment (waldyrious) [inline pages/linux/repquota.md]: I'd be specific here and say "used quota space", or just "used quota"

File: pages/linux/repquota.md (hunk 3 above)
Comment (pearcem0) [inline pages/linux/repquota.md]: How about - 
'Report on quotas for all users (unless specified); only super-user may view quotas which are not their own:' ?

File: pages/linux/repquota.md (hunk 5 above)
Comment (pearcem0) [inline pages/linux/repquota.md]: okay, will do - Thanks!

File: pages/linux/repquota.md (hunk 3 above)
Comment (waldyrious) [inline pages/linux/repquota.md]: So this command gives an error if not run with sudo and no users are specified? How about including sudo in its invocation?

**Comment by agnivade** [comment]
Hi @pearcem0, any updates on this ?

File: pages/linux/repquota.md (hunk 3 above)
Comment (pearcem0) [inline pages/linux/repquota.md]: When I say default, I mean running sudo repquota {{user}} will do the same thing as sudo repquota -u {{user}}. 
Okay, how about having two entries, one for reporting quota on your own user, and one for reporting quota on other users (as root) - 

- Report on quotas for current user:

`repquota --user {{current_user}}`

- Report on quotas for another user:

`sudo repquota --user {{another_user}}`

- Report on quotas for user groups:

`sudo repquota --group {{group}}`

I also added sudo for group. 

The command itself is generally an administrators tool, only the super-user may view quotas which are not their own. Running without sudo generates a permissions error. 
Should I add sudo to all of the commands?

File: pages/linux/repquota.md (hunk 4 above)
Comment (pearcem0) [inline pages/linux/repquota.md]: Report quota, even for users those who do not consume resources. How about - 
Verbose mode - report all quotas, even for users those who do not consume resources:

**Comment by pearcem0** [comment]
Hi @agnivade - So sorry for the delay. I've added some comments in reply to waldyrious so that we can continue.

File: pages/linux/repquota.md (hunk 4 above)
Comment (waldyrious) [inline pages/linux/repquota.md]: So the quotas reported won't be the limits for each user, but actual usages that they make of their quota? Or both?

File: pages/linux/repquota.md (hunk 3 above)
Comment (waldyrious) [inline pages/linux/repquota.md]: I think it makes sense to separate different usages, yes, and to add sudo to all commands that require it (except those related to viewing one's own quota)

File: pages/linux/repquota.md (hunk 4 above)
Comment (pearcem0) [inline pages/linux/repquota.md]: Both.  Used (usage), soft limit, hard limit, time left on the quota timer etc.

Comment (waldyrious) [inline pages/linux/repquota.md]: Ok, so how about "Report quota stats for all users, even those who aren't using any of their quota"?

File: pages/linux/repquota.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# repquota
+
+> Display a summary of existing file quotas for a filesystem.
+
+- Report on all existing quotas:
```
Comment (waldyrious) [inline pages/linux/repquota.md]: In line with the suggested description below, this could be "Report stats for all quotas in use:"?

**Comment by waldyrious** [comment]
Alright! I think we've given you enough headaches @pearcem0 😄 thanks so much for your patience! LGTM 👍

**Pull request merged**